### PR TITLE
document  `COV_EXCL_START`, `COV_EXCL_STOP` and `COV_EXCL_LINE`

### PR DESCRIPTION
**Pull request by t-bltg**
Fix https://github.com/JuliaCI/Coverage.jl/issues/326.

@IainNZ, @fingolfin or @kshyatt, bump ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,20 @@ Be aware of a few limitations:
 - a line that can take one of two branches gets marked as covered even if only one branch is tested
 - currently, code run by Julia's internal interpreter [is not marked as covered](https://github.com/JuliaLang/julia/issues/37059).
 
+### Exclude specific lines or sections from coverage
+
+To exclude specific sections, wrap the section in `COV_EXCL_START` - `COV_EXCL_STOP` blocks:
```
Comment (fingolfin) [inline README.md:69]: In how far are these "blocks"? Maybe "sandwich" the section between such "comments"? Or "surround ... by"? Or "demarcate"

File: README.md [hunk 2]
```diff
@@ -64,6 +64,20 @@ Be aware of a few limitations:
...
+```julia
+# COV_EXCL_START
+foo() = nothing
+# COV_EXCL_STOP
+```
+
+To exclude a single line, add `COV_EXCL_LINE` anywhere:
```
Comment (fingolfin) [inline README.md:76]: "Anywhere"? Perhaps "add a comment with COVL_EXCL_LINE on that line"?

File: README.md (hunk 1 above)
Comment (t-bltg) [reply README.md:69]: `wrap` is pretty obvious to me, but if this is unclear I'll rephrase that.

File: README.md (hunk 2 above)
Comment (t-bltg) [reply README.md]: Anywhere was for the `occursin` https://github.com/JuliaCI/CoverageTools.jl/blob/9d8fba371f4c65a7e9ca3bdaae753a52bcb22218/src/CoverageTools.jl#L212

Comment (t-bltg) [reply README.md]: I mean:
```julia
const #=COVL_EXCL_LINE=# a = 1
```
would also work, as awkward as it may be.

**Comment by kshyatt** [comment]
LGTM. Any barrier to merging this?

**Comment by vtjnash** [comment]
Maybe should be over on CoverageTools.jl, since this is not a feature here anymore?

**Pull request merged**